mmv1/templates/terraform/examples/cloudfunctions2_basic.tf.erb

```diff
@@ -1,7 +1,15 @@
 # [START functions_v2_basic]
+locals {
```

Comment:
do we need this `locals` defined here? i think we could remove this and just refer to the variable directly on line 12. i also don't think we'll need to define the provider block below (line 6), that should be handled by marking this example as min_version: beta i believe

Reply:
The `locals` and provider block are necessary for the tutorial to be frictionless.